hack/e2e/run.sh

```diff
@@ -47,7 +47,7 @@ UP="${UP:-yes}"
 # if DOWN==yes, delete cluster after test
 DOWN="${DOWN:-yes}"
 
-KUBERNETES_VERSION="${KUBERNETES_VERSION:-v1.23.2}"
+KUBERNETES_VERSION="${KUBERNETES_VERSION:-v0.24.0-alpha.3}"
```

Comment:
should it be 1.24?

Reply:
```

I0318 22:24:39.156681   25487 populate_instancegroup_spec.go:308] Cannot set default Image for CloudProvider="aws"
Error: unable to determine default image for InstanceGroup master-us-west-2a
Error: exit status 1 
```
does kops support creating such clusters yet?

Reply:
Yeah I was wondering if that would work :)